build.gradle

```diff
@@ -79,7 +79,7 @@ dependencies {
     // This dependency is found on compile classpath of this component and consumers.
     compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.3.4'
     compile 'com.google.guava:guava:23.0'
```

Comment:
If possible try switching these compiles to implementations, it provides better encapsulation but more importantly runs faster on compile. https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle

Reply:
done